src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1473,7 +1474,8 @@ static private DynamoPreferencesData DynamoPreferencesDataFromJson(string json)
                 },
                 ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
                 TypeNameHandling = TypeNameHandling.Auto,
-                Formatting = Newtonsoft.Json.Formatting.Indented
+                Formatting = Newtonsoft.Json.Formatting.Indented,
```

Comment:
not required - but did you consider pulling out these settings to some util function as these all seem the same?